packages/provider/src/speech-model/v3/speech-model-v3-call-options.ts

```diff
@@ -1,6 +1,6 @@
-import { JSONValue } from '../../json-value/json-value';
+import { JSONObject } from '../../json-value/json-value';
 
-type SpeechModelV3ProviderOptions = Record<string, Record<string, JSONValue>>;
+type SpeechModelV3ProviderOptions = Record<string, JSONObject>;
```

Comment:
ideally we'd use shared options (also fro transcription) but that could be for separate prs